package.json

```diff
@@ -23,7 +23,7 @@
     "url": "git+https://github.com/twilio/twilio-video-processors.js.git"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=18"
```

Comment:
Can we mention the minimum version requirement in the changelog?

Reply:
Good catch! I just updated the changelog to mention it